GameOfLifeApp/GameManager.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using GameOfLifeApp;
+
+namespace GameOfLifeApp
+{
+    /// <summary>
+    /// Starts iteration.
+    /// </summary>
+    public class GameManager
+    {
+        public GameLogic? data;
+        public Communicator? communicator;
+        public Display? display;
+        public Serializer? serializer;
```

Comment:
need to inject all the logic classes not create them in the constructor or a method, and classes are reference types so no need to make them nullable, you can google what are reference types and what are value types to make it more understandable hopefully

Reply:
Done.